index.html

```diff
@@ -816,18 +817,29 @@ <h3>
             This can be useful for analytics and possibly other customizations.
             However, it is also conceivable that developers could encode
             strings into the start_url that uniquely identify the user (e.g., a
-            server assigned <abbr>UUID</abbr>). This is fingerprinting/privacy
-            sensitive information that the user might not be aware of.
+            server-assigned <abbr>UUID</abbr> such as `"?user=123"`,
+            `"/user/123/"`, or `"https://user123.foo.bar"`). This is
+            fingerprinting/privacy sensitive information that the user might
+            not be aware of.
+          </p>
+          <p class="note">
+            It would be irresponsible for a developer to use the [=start URL=]
+            to include information that uniquely identifies a user, as it would
+            represent a fingerprint that is not cleared when the user clears
+            site data. However, nothing in this specification can practically
+            prevent developers from doing this.
```

Comment:
```suggestion
            prevent developers from doing this. Similarly, nothing in this specification prevents user agents from identifying 
            and stripping out unique identifiers start URLs.
```

Reply:
Maybe we should just make the above some may... that the user agent MAY strip out any personally identifiable information or IDs from start URLs.

Reply:
Hmm.... I can't exactly point to the bit that says that you can't do this, but it would be weird I think if user agents do this. (i.e. it would break sites - you can't just randomly delete any single character from a URL because you don't know if it's important.)

The top of this spec says "The start_url member is purely advisory" so I guess UAs can do whatever they want. But I think there is an expectation that UAs actually load up the app at the URL developers specify and they shouldn't be changing it. (I would perhaps suggest we remove the statement "The start_url member is purely advisory".)

So while this may technically be true, I wouldn't say it like this as if it's a good idea for UAs to do.